tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -51,7 +51,13 @@ def test_check_sfputil_presence(duthosts, enum_rand_one_per_hwsku_frontend_hostn
     portmap, dev_conn = get_dev_conn(duthost, conn_graph_facts, enum_frontend_asic_index)
 
     logging.info("Check output of '{}'".format(cmd_sfp_presence))
-    sfp_presence = duthost.command(cmd_sfp_presence)
+    sfp_presence = duthost.command(cmd_sfp_presence, module_ignore_errors=True)
```

Comment:
@yutongzhang-microsoft I see the addition of `module_ignore_errors=True` as a common case. How is that going to impact the test if it ran on physical h/w and the command failed ? Do you think the parameter is suited only for `vs` and not otherwise ?

Reply:
Hi, @opcoder0, If the command failed, the rc code will not be 0. 
The code `assert sfp_presence['rc'] == 0, "Run command '{}' failed".format(cmd_sfp_presence)` will handle this situation.